src/components/Comment/Comment.tsx

```diff
@@ -168,14 +164,23 @@ const regularFont = css`
 const svgOverrides = css`
 	svg {
 		fill: ${neutral[46]} !important;
-		left: 3px !important;
-		bottom: 0 !important;
-		width: 20px !important;
-		height: 20px !important;
-		margin: 0px !important;
 	}
 `;
 
+const linkOverrides = css`
+	a {
+		display: flex;
+		flex-direction: row;
+		align-items: inherit;
+	}
+`
+
+const buttonOverrides = css`
+	.src-button-space {
+		width: 0px !important;
+	}
```

Comment:
This coupling to classnames from source with !important on top probably isn't something we want to do. Maybe we can just accept the way Souorce does things?